src/Symfony/Component/DependencyInjection/ExpressionLanguageProvider.php

```diff
@@ -38,6 +39,20 @@ public function getFunctions()
             }, function (array $variables, $value) {
                 return $variables['container']->getParameter($value);
             }),
+
+            new ExpressionFunction('env', function ($arg, $default = null) {
+                if (2 > func_num_args()) {
+                    return sprintf('Symfony\\Component\\DependencyInjection\\Environment::getVariable(%s)', $arg);
+                }
+
+                return sprintf('Symfony\\Component\\DependencyInjection\\Environment::getVariable(%s, %s)', $arg, $default);
+            }, function (array $variables, $value, $default = null) {
```

Comment:
Instead of the Environment class, my personal preference to avoid a public method would to rebind this closure.
`\Closure::bind(..., null, Container::class)`